metadata_ingest/form_parsers.py

```diff
@@ -100,8 +99,8 @@ def parse_fields(self, fields):
         parsed_fields['bureauCode'] = ",".join(self.clean_comma_delim(parsed_fields['bureauCode']))
         parsed_fields['programCode'] = ",".join(self.clean_comma_delim(parsed_fields['programCode']))
         parsed_fields['keyword'] = self.clean_comma_delim(parsed_fields['keyword'])
-        parsed_fields['identifiersExtended'] = self.parse_identifiersExtended(parsed_fields['identifiersExtended'])
-        parsed_fields['contactPoint'] = self.parse_contactPoints(parsed_fields['contactPoint'])
+        parsed_fields['identifiersExtended'] = self.parse_identifiers_extended(parsed_fields['identifiersExtended'])
+        parsed_fields['contactPoints'] = self.parse_contact_points(parsed_fields['contactPoints'])
```

Comment:
changes to the dictionary key here should be reverted (`contactPoints` is not an expected key in the dictionary)